data/net/minecraft/client/User.mapping

```diff
@@ -1,4 +1,11 @@
 CLASS net/minecraft/client/User
+	METHOD <init> (Ljava/lang/String;Ljava/lang/String;Ljava/lang/String;Ljava/lang/String;)V
+		ARG 1 name
+		ARG 2 uuid
+		ARG 3 accessToken
+		ARG 4 sessionType
```

Comment:
`sessionType` -> `typeName`

Reply:
So the argument is passed into `Type#byName(String sessionType)`, which already had the `sessionType` name before I touched the class, would it not be inconsistent to use anything but `sessionType` since it is simply passed into that method?